content/guides/component-testing/faq.md

```diff
@@ -0,0 +1,192 @@
+developer tools that you are used to when building web applications, such as
+element inspectors, modifying CSS, and source debugging.
+
+Cypress Component Testing is built around the same tools and APIs that
+end-to-end testing uses. Anyone familiar with Cypress can immediately hop in and
+feel productive writing component tests without a large learning curve.
+Component tests can also use the vast Cypress ecosystem, plugins, and services
+(like [Cypress Dashboard](https://www.cypress.io/dashboard) already available to
+complement your component tests.
+
+## Technical Questions
+
+### What is the Mount Function?
+
+We ship a `mount` function for each UI library that is imported from the
+`cypress` package. It is responsible for rendering components within Cypress's
+sandboxed iframe and handling any framework-specific cleanup.
+
+```js
+//example showing importing mount command in react
+import { mount } from 'cypress/react'
+```
+
+While you can use the `mount` function in your tests, we recommend using
+[`cy.mount()`](/api/commands/mount), which is added as a
+[custom command](/api/cypress-api/custom-commands) in the
+**cypress/support/component.js** file:
+
+<code-group>
+<code-block label="cypress/support/component.js" active>
+
+```ts
+import { mount } from 'cypress/react'
+
+Cypress.Commands.add('mount', mount)
+```
+
+</code-block>
+</code-group>
+
+This allows you to use `cy.mount()` in any component test without having to
+import the framework-specific mount command, as well as customizing it to fit
+your needs. See the examples guide for each framework for info on creating a
+custom `cy.mount()` command.
+
+### Why isn't my component rendering as it should?
+
+Any global styles and fonts must be imported and made available to your
+component, just like in the application. See our guide on
+[Styling Components](/guides/component-testing/styling-components) for more
+information on doing so.
+
+### Why doesn't my spec show in the spec explorer?
+
+If something appears missing from the spec list, make sure the files have the
+[proper extension and the `specPattern` is correctly defined](/guides/component-testing/component-framework-configuration#Spec-Pattern-for-Component-Tests).
+
+### How do I fix ESLint errors for things like using the global Cypress objects?
+
+If you experience ESLint errors in your code editor around Cypress globals,
+install the
+[`eslint-plugin-cypress`](https://www.npmjs.com/package/eslint-plugin-cypress)
+ESLint plugin.
+
+### Why isn't TypeScript recognizing the global Cypress objects or custom cypress commands (eg: `cy.mount`)?
+
+In some instances, TypeScript might not recognize the custom `cy.mount()`
+command in Cypress spec files not located in the **cypress** directory. You will
+get a compiler error specifying that the type is not found in this case.
+
+A quick way to fix this is to include the **\_**cypress**\_** directory in your
```

Comment:
`cypress` and `tsconfig.json` formatted correctly here? not sure what the `_`s are supposed to do

Reply:
nice catch, not sure what that was about